apps/classroomio-com/src/app.css

```diff
@@ -1,3 +1,5 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
```

Comment:
why are we importing the inter css

Reply:
Usman changed the font of the hero section to inter so i had to import it and make it a class to reuse

Reply:
let's use the font that we used to have. No need for inter

Reply:
alright, noted